external/webserver/http_client.c

```diff
@@ -305,6 +339,9 @@ int http_parse_message(char *buf, int buf_len, int *method, char *url,
 						if (strcmp(key, "Sec-WebSocket-Key") == 0) {
 							strncpy((char *)client->ws_key, value, WEBSOCKET_CLIENT_KEY_LEN);
 						}
+						if (strcasecmp(key, "Keep-alive") == 0) {
```

Comment:
And it is better to check the length for comparision, for example `strncasecmp` instead of `strcasecmp`.

Reply:
'\0' is compared using strcasecmp